pkg/ebpf/events_processor.go

```diff
@@ -282,75 +247,6 @@ func (t *Tracee) processSchedProcessExec(event *trace.Event) error {
 	return nil
 }
 
-func (t *Tracee) processSchedProcessExit(event *trace.Event) error {
-	if !t.config.ProcessInfo {
-		return nil
-	}
-	if t.config.Capture.NetPerProcess {
-		pcapContext, _, err := t.getPcapContextFromTid(uint32(event.HostThreadID))
-		if err == nil {
-			go t.netExit(pcapContext)
-		}
-	}
-
-	go t.deleteProcInfoDelayed(event.HostThreadID)
-	return nil
-}
-
-func (t *Tracee) processSchedProcessFork(event *trace.Event) error {
-	err := t.convertArgMonotonicToEpochTime(event, "start_time")
-	if err != nil {
-		return err
-	}
```

Comment:
This code is required, unrelated to procinfo

Reply:
The only place it was used was within the procinfo.ProcessCtx{} (will be worked together with https://github.com/aquasecurity/tracee/pull/2569#discussion_r1070620792)

Reply:
![image](https://user-images.githubusercontent.com/7395852/212728300-521f6c80-37f7-49a9-b420-5344f1b27b7d.png)

Reply:
Added a reminder to bring logic back in with the refactor.

Reply:
This logic was added in this PR: https://github.com/aquasecurity/tracee/pull/2387 and is required. Why do you want to remove it?

Reply:
I missed seeing it had a pointer to Event as an argument, and that it would update the ts of the given argument (thought it was only for the proc info work being done bellow). I have put it back in place.